pinot-core/src/main/java/com/linkedin/pinot/core/segment/store/SegmentLocalFSDirectory.java

```diff
@@ -75,7 +75,34 @@ public Path getPath() {
 
   @Override
   public long getDiskSizeBytes() {
-    return FileUtils.sizeOfDirectory(segmentDirectory.toPath().toFile());
+    // [PINOT-3479] For newly added refresh segments, the new segment will
```

Comment:
Talked to @raviarin  in-person. He is fine with this. In future, we need to consider making our tickets public.